app/src/main/java/org/simple/clinic/monthlyReports/Questionnaire.kt

```diff
@@ -0,0 +1,18 @@
+package org.simple.clinic.monthlyReports
+
+import android.os.Parcelable
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import kotlinx.parcelize.Parcelize
+import java.util.UUID
+
+@Entity(tableName = "Questionnaire")
+@Parcelize
+data class Questionnaire(
+    @PrimaryKey
+    val uuid: UUID,
+
+    val questionnaire_type: QuestionnaireType,
```

Comment:
I think we discussed of making `QuestionnaireType` as primary key in PRD. Please check

Reply:
Primary key update to `questionnaire_type`